pkg/controller/dataimportcron-controller.go

```diff
@@ -651,6 +661,24 @@ func (r *DataImportCronReconciler) newCronJob(cron *cdiv1.DataImportCron) (*v1be
 	return cronJob, nil
 }
 
+func (r *DataImportCronReconciler) checkFailingDataImportCrons() error {
+	numNotUpToDate := 0
+	dataImportCronList := &cdiv1.DataImportCronList{}
+	if err := r.client.List(context.TODO(), dataImportCronList); err != nil {
+		return err
```

Comment:
what about IsNotFound(err)? I think you should return nil in that case

Reply:
Then we will get an empty list and thus set the metric value to 0 which is what we want (back off with nil means we won't update the metric at all)